__std__/cells/plutus/library/agda-packages.nix

```diff
@@ -36,7 +36,7 @@ let
     {
       haskellPackages = pkgs.haskellPackages //
       {
-        ghcWithPackages = cell.library.plutus-project.ghcWithPackages;
+        ghcWithPackages = (cell.library.plutus-project { }).ghcWithPackages;
```

Comment:
I'd love to be able to get rid of these `(cell.library.plutus-project { })`
To do that, we can
1. Rename `__std__/cells/plutus/library/plutus-project.nix` -> `__std__/cells/plutus/library/make-plutus-project.nix` 
2. Create `__std__/cells/plutus/library/plutus-project-ghc8107.nix`
    ```
    { inputs, cell }:
    cell.library.make-plutus-project { compiler-nix-name = "ghc8107" } 
    ```
3. Create `__std__/cells/plutus/library/plutus-project-ghc942.nix`
    ```
    { inputs, cell }:
    cell.library.make-plutus-project { compiler-nix-name = "ghc942" } 
    ```

Reply:
It would also make it very explicit where we are using which

Reply:
Then we could also get rid of the file `__std__/cells/plutus/library/ghc-compiler-nix-name.nix`

Reply:
Hmmmm, I guess that's fine yes.